build/patch/pre-autogen/libevent/common/000.patch

```diff
@@ -108,19 +79,49 @@ index 284341c..5b8b0a6 100644
 + * have removed arc4random_addrandom() since 1.26. Since then, other libcs
 + * followed suit (e.g. Android's own libc). But libevent's arc4random.c
 + * copy still implement arc4random_addrandom().
++ *
++ * See also:
++ *
++ * - https://github.com/measurement-kit/libevent/commit/8b275d967d7ffd95d5cc12069aef35669126c6d9
++ * - https://bugzilla.mozilla.org/show_bug.cgi?id=931354
++ * - https://bug931354.bmoattachments.org/attachment.cgi?id=829728
 + */
-+#define _EVENT_HAVE_ARC4RANDOM_ADDRANDOM 1
++#ifndef EVENT__HAVE_ARC4RANDOM_ADDRANDOM
++#define EVENT__HAVE_ARC4RANDOM_ADDRANDOM 1
++#endif
 +
- #endif /* } !_EVENT_HAVE_ARC4RANDOM */
+ #endif /* } !EVENT__HAVE_ARC4RANDOM */
  
  void
-@@ -174,7 +186,8 @@ evutil_secure_rng_get_bytes(void *buf, size_t n)
+@@ -195,8 +213,10 @@ evutil_secure_rng_get_bytes(void *buf, size_t n)
  void
  evutil_secure_rng_add_bytes(const char *buf, size_t n)
  {
-+#if defined _EVENT_HAVE_ARC4RANDOM_ADDRANDOM
++#if defined EVENT__HAVE_ARC4RANDOM_ADDRANDOM
```

Comment:
@AntonioLangiu Here, basically, the name has been changed to follow libevent-2.1 convention where underscore names are no longer used for macro variables.